project/emscripten/makefile

```
@@ -15,6 +15,43 @@ multiThreaded: export EXTRA_CPP_FLAGS := -DBOXEDWINE_MULTI_THREADED -pthread
+#   -sALLOW_TABLE_GROWTH     – allows addFunction() to grow the wasmTable
+#   -sALLOW_MEMORY_GROWTH    – compiled blocks can allocate their own memory
+#   -sMAXIMUM_MEMORY=3 GiB   – long sessions accumulate many compiled
+#                              blocks; without an explicit max,
+#                              ALLOW_MEMORY_GROWTH caps near 2 GiB.
+jit: export BUILD_DIR := Build/Jit
+jit: export EXTRA_CPP_FLAGS := -DBOXEDWINE_WASM_JIT
+jit: export SHELL_FILE := shell.html
+# _malloc/_free + HEAPU32: used by boxedwine-shell.js (finishGroupRelocPatching,
+# called from onRuntimeInitialized) to allocate and zero the per-entry reloc
+# slot arrays patched into grouped cache modules.
+jit: export EXTRA_LD_FLAGS := -sALLOW_TABLE_GROWTH -sALLOW_MEMORY_GROWTH -sMAXIMUM_MEMORY=3221225472 -sEXPORTED_RUNTIME_METHODS=addFunction,removeFunction,wasmTable,HEAPU32 -sEXPORTED_FUNCTIONS=_main,_wasm_jit_import_from_file,_wasm_jit_set_persistence_active,_wasm_jit_set_record_active,_malloc,_free
```

checkmake (0.3.2)

[warning] Target "jit" defined multiple times (lines 28 and 34).

(uniquetargets)